src/components/ResultsList.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+
+class ResultsList extends Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			results: [],
+		};
+		
+		this.displayList = this.displayList.bind(this);
+		this.tableSort = this.tableSort.bind(this);
+	}
+
+	componentWillReceiveProps(nextProps) {
+		this.setState({ results: nextProps.results });
+	}
+
+	displayList() {
+		if (this.state.results.length === 0) {
+			return (
```

Comment:
I would consider not showing a table at all if there are no results and just making this a p